code/modules/projectiles/gun.dm

```diff
@@ -46,7 +46,7 @@
 /obj/item/weapon/gun/examine(mob/user)
 	..()
 	if(pin)
-		user << "It has a [pin] installed."
+		user << "It has [pin] installed."
```

Comment:
it just hit me how exponentially worse that would make everything else. nevermind then, better than usual byond grammar anyway.